src/appkit.cs

```diff
@@ -18378,11 +18379,17 @@ partial interface NSWindow : NSAnimatablePropertyContainer, NSUserInterfaceItemI
 	
 		[Export ("contentRectForFrameRect:")]
 		CGRect ContentRectFor (CGRect frameRect);
-	
+
+		[Export ("init")]
+		[PostSnippet ("if (!DisableReleasedWhenClosedInConstructor) { ReleasedWhenClosed = false; }")]
```

Comment:
If they set DisableReleasedWhenClosedInConstructor = true I didn't want to change the state of the variable, in case it was already false.

The use case I knew of is setting ReleasedWhenClosed = true, but I'm not sure if there are cases where it case be false.